tools/webdriver/webdriver/bidi/__init__.py

```diff
@@ -0,0 +1 @@
+from .client import BidiSession
```

Comment:
flake8 is unhappy about this unused import. It looks like this is so that both `webdriver.bibi.BidiSession` and `webdriver.bibi.client.BidiSession` can be used, but is that a good idea? (Both variants do seem to be use in the code.)

Reply:
I think allowing the former is a good quality of life improvement, and exporting the main functionality at the package level is common. I can silence the flake8 warning.